feat(components): create admin page

Hellomyguest opened the pull request:
META DATA: issue22

Commit:
feat(components): create admin page

META DATA: issue22

Commit:
fix(components): change space direction for datepickers

META DATA: issue22

dpolevodin commented on src/pages/admin/page.tsx:
```diff
@@ -1,11 +1,379 @@
+    return () => deleteAdditionalDateFn(dateToDelete);
+  };
+
+  console.log(blocks);
```
не забыть убрать console.log

Hellomyguest replied:
done

dpolevodin commented on src/pages/admin/page.tsx:
```diff
@@ -1,11 +1,379 @@
+            labelCol={{ span: 8 }}
+            wrapperCol={{ span: 24 }}
+            initialValues={{ remember: true }}
+            onFinish={(value) => addTutorFn(value)}
```
предлагаю переписать тип addTutorFn, чтобы убрать стрелочную функцию из пропса

Hellomyguest replied:
done

dpolevodin commented on src/pages/admin/page.tsx:
```diff
@@ -1,11 +1,379 @@
+              labelCol={{ span: 8 }}
+              wrapperCol={{ span: 24 }}
+              initialValues={tutor}
+              onFinish={(value) => deleteTutorFn(value.telegramNickName)}
```
предлагаю переписать тип deleteTutorFn, чтобы убрать стрелочную функцию из пропса

Hellomyguest replied:
done

dpolevodin commented on src/pages/admin/page.tsx:
```diff
@@ -1,11 +1,379 @@
+            labelCol={{ span: 8 }}
+            wrapperCol={{ span: 24 }}
+            initialValues={{ remember: true }}
+            onFinish={(value) => addRepositoryFn(value)}
```
предлагаю переписать тип addRepositoryFn, чтобы убрать стрелочную функцию из пропса

Hellomyguest replied:
done

dpolevodin commented on src/pages/admin/page.tsx:
```diff
@@ -1,11 +1,379 @@
+              labelCol={{ span: 8 }}
+              wrapperCol={{ span: 24 }}
+              initialValues={repository}
+              onFinish={(value) => deleteRepositoryFn(value.name)}
```
предлагаю переписать тип deleteRepositoryFn, чтобы убрать стрелочную функцию из пропса

Hellomyguest replied:
done

dpolevodin commented on src/pages/admin/page.tsx:
```diff
@@ -1,11 +1,379 @@
+            labelCol={{ span: 8 }}
+            wrapperCol={{ span: 24 }}
+            initialValues={{ remember: true }}
+            onFinish={(value) => addBlockFn(value)}
```
предлагаю переписать тип addBlockFn, чтобы убрать стрелочную функцию из пропса

Hellomyguest replied:
done

dpolevodin commented on src/pages/admin/page.tsx:
```diff
@@ -1,11 +1,379 @@
+              name={block.name}
+              labelCol={{ span: 8 }}
+              wrapperCol={{ span: 24 }}
+              onFinish={(value) => deleteBlockFn(value.blockName)}
```
предлагаю переписать тип deleteBlockFn, чтобы убрать стрелочную функцию из пропса

Hellomyguest replied:
done

Commit:
fix(components): fix handle functions for components

META DATA: issue22

makhnatkin commented on src/pages/admin/model.ts:
```diff
@@ -0,0 +1,90 @@
+
+export const $tutors = createStore([
+  {
+    fullName: "Махнаткин Сергей",
```
add TODO or FIXME comment

makhnatkin commented on src/pages/admin/model.ts:
```diff
@@ -0,0 +1,90 @@
+$tutors
+  .on(addTutor, (state, payload) => [...state, payload])
+  .on(deleteTutor, (state, payload) =>
+    state.filter((tutor) => tutor.telegramNickName !== payload)
```
why filter by telegramNickName?
use id field

Commit:
feat(components): add AddForm and StoreDisplay components

AddForm is used to add data to store
StoreDisplay display the information from provided store

META DATA: issue22

Commit:
feat(components): add AddDate and StoreDisplayDates components

META DATA: issue22

makhnatkin commented on package.json:
```diff
@@ -29,6 +30,7 @@
     "@vitejs/plugin-react": "^2.2.0",
     "commitizen": "^4.2.5",
     "cz-customizable": "^7.0.0",
+    "effector-logger": "^0.13.4",
```
devDeps?

makhnatkin commented on src/features/add-date/AddDate.tsx:
```diff
@@ -0,0 +1,18 @@
+import "./AddDate.css";
+
+type Props = {
+  handleChangeAddDate: (...args: any[]) => void;
```
not use any
